README.md

```diff
@@ -1 +1,84 @@
-# node-process
\ No newline at end of file
+# Node Process Object
+
+## Overview
+
+This lesson will cover the `process` global object in Node to get system information and manipulate with the Node instance. 
+
+## Objectives
+
+1. Describe the process objects
+1. Describe about process information such as versions, pid, cwd, features, uptime, memoryUsage
+1. Describe how to exit the process
+1. Describe environment variables
+2. Describe standard input and output
+
+## Process Interface
+
+`process` has a lengthy number of properties related to the currently running Node instance and the environment. 
+
+* `process.env`: Environment variables
+* `process.pid`: Process ID 
+* `process.platform`: Platform, e.g., `darwin`
+* `process.cwd()`: Current working directory. Not always the same as `__dirname`.
+* `process.version`: Version of node
+* `process.versions`: Versions of node, V8, zlib and other Node internal components
+* `process.features`: Features of this Node instance, e.g., `debug`p
+* `process.arch`: Architecture of this system, e.g., `x64`
+* `process.uptime()`: Time this process runs in seconds
+* `process.memoryUsage()`: The heap total and used numbers
+
+## Exiting The Process
+
+It also has methods to terminate the process:
+
+* `process.exit(1)`: Exit the current process with errors
+* `process.exit(0)`: Exit the current process with no errors
+* `process.kill(pid)`: Terminate a process with ID `pid`, e.g., to kill self is `process.kill(process.pid)`
```

Comment:
need to include what a `pid` is . whats the different between killing and exiting?